### ISSUE TITLE
Add example of options

### ISSUE THREAD

**Issue by crystalfp** [neocotic/html.md#63]
A very small request. On the API page (http://neocotic.com/html.md/#api) add an example of md called with options. Eg. `md(myHTML, {inline: 1})` (BTW, I presume this is correct. At least it works in my code).
Thanks for the good work!
mario

**Comment by neocotic** [comment]
Good idea. I'll try to do this and update the README as well. What you have should work, although I'd recommend sticking to correct data types (boolean `true` vs truthy values).

**Comment by crystalfp** [comment]
Thanks Alasdair!


On 02-Jan-16 15:32, Alasdair Mercer wrote:
> Good idea. I'll try to do this and update the README as well. What you
> have should work, although I'd recommend sticking to correct data types
> (boolean |true| vs truthy values).
>
This was exactly my problem. A quick look at the code convinced me that 
I have to use a numerical 1 and not 'true' (in retrospect is obvious I 
have to use the boolean value).

Thanks again for your wonderful piece of code!
mario
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/neocotic/html.md/issues/63#issuecomment-168396297>.
>

-- 
Ing. Mario Valle
Swiss National Supercomputing Centre (CSCS)
v. Trevano 131, 6900 Lugano, Switzerland
Tel: +41 (91) 610.82.60
